pom.xml

```diff
@@ -13,7 +13,7 @@
 
   <groupId>org.datadog.jenkins.plugins</groupId>
   <artifactId>datadog</artifactId>
-  <version>0.3.0-SNAPSHOT</version>
+  <version>0.3.1-SNAPSHOT</version>
```

Comment:
This is handled elsewhere in our, so this change isn't necessary at this step; go ahead and remove it.